coderd/util/time/duration.go

```diff
@@ -0,0 +1,44 @@
+package duration
+
+import (
+	"fmt"
+	"time"
+)
+
+type Unit struct {
+	value int
+	unit  string
+}
+
+func Humanize(d time.Duration) string {
```

Comment:
Did you consider using https://pkg.go.dev/github.com/dustin/go-humanize#RelTime? We already use this library.

Reply:
Yes, but I was unsure if we wanted to have this humanized like this much or wanted something more descriptive like "X days, Y hours, and W minutes" or "around X days". Since it is a time operation I think the first could be best but I like both options honestly. What do you think?

Reply:
Instead of answering you directly, let me answer with another question:
As a user, do you care that your workspace is going to be deleted in "23 hours 59 minutes and 42 seconds" or "around 1 day"? Is the precision of the former case useful in some way?

Reply:
Good question, not really 😓 . Going to fix that.